### PR TITLE
Translations for the labels taxonomy

### PR DESCRIPTION
**Pull request by teolemon**
Translations for the labels taxonomy
Note: some translations on xx:Labels

### CODE REVIEW

File: taxonomies/labels.txt [hunk 1]
```diff
@@ -531,12 +539,15 @@ ru:ГОСТ Р ИСО 9001-2008, ГОСТ ISO 9001-2011
 ru:ГОСТ Р ИСО 9001-2015
 
 en:ISO 22000
+it:ISO 22000
+pl:ISO 22000
```
Comment (teolemon) [inline taxonomies/labels.txt:543]:
```suggestion
```

File: taxonomies/labels.txt [hunk 2]
```diff
@@ -18599,5 +19034,8 @@ en:Nestlé Cocoa Plan, Cocoa Plan
 en:EU Ecolabel
 xx:EU Ecolabel
 
+<<<<<<< Updated upstream
```
Comment (stephanegigandet) [inline taxonomies/labels.txt:19037]: this looks like an unresolved conflict

File: taxonomies/labels.txt [hunk 3]
```diff
@@ -18599,5 +19034,8 @@ en:Nestlé Cocoa Plan, Cocoa Plan
...
 en:GLOBALG.A.P.
 xx:GLOBALG.A.P., Global GAP
+=======
+>>>>>>> Stashed changes
```
Comment (teolemon) [inline taxonomies/labels.txt:19041]:
```suggestion
en:GLOBALG.A.P.
xx:GLOBALG.A.P., Global GAP
```

File: taxonomies/labels.txt (hunk 2 above)
Comment (teolemon) [reply taxonomies/labels.txt]: fixed

File: taxonomies/labels.txt [hunk 4]
```diff
@@ -2730,13 +2784,15 @@ opposite:en: en:No GMOs
...
 ca:NON GMO Project
+es:Proyecto no OGM
 fi:Non GMO project, Ei-GMO-projekti
 it:Progetto non OGM
 pt:Non GMO project, Projeto não OGM
 wikidata:en:Q7754286
 
 <en:No GMOs
 en:Ohne Gentechnik
+cs:Označení &quot;Ohne Gentechnik&quot;
```
Comment (teolemon) [inline taxonomies/labels.txt:2795]: TOFIX

File: taxonomies/labels.txt [hunk 5]
```diff
@@ -3269,11 +3343,11 @@ es:Comité de Agricultura Ecológica de la Comunitat Valenciana, comite agricult
 country:en:Spain
 
 <en:Organic
-es:Consejo de Agricultura Ecologica de la Región de Murcia, comite agricultura ecologica comunidad region de murcia, CAERM
-country:en:Spain
 <en:Organic
+es:Consejo de Agricultura Ecologica de la Región de Murcia, comite agricultura ecologica comunidad region de murcia, CAERM
 es:Comite Andaluz de Agricultura Ecologica, CAAE
 country:en:Spain
+country:en:Spain
```
Comment (teolemon) [inline taxonomies/labels.txt:3350]:
```suggestion
country:en:Spain
```
Duplicate during merge @stephanegigandet

**Review by stephanegigandet — APPROVED** [review]

File: taxonomies/labels.txt (hunk 4 above)
Comment (teolemon) [inline taxonomies/labels.txt:2795]:
```suggestion
cs:Ohne Gentechnik
```

File: taxonomies/labels.txt (hunk 5 above)
Comment (teolemon) [inline taxonomies/labels.txt:3350]:
```suggestion
country:en:Spain
```

Comment (teolemon) [inline taxonomies/labels.txt:3350]:
```suggestion
country:en:Spain
```

**Pull request merged**